origins_workflow/origins_workflow.module

```diff
@@ -166,6 +167,48 @@ function origins_workflow_entity_update(EntityInterface $entity) {
       }
     }
   }
+  // Now add processing for moderation sidebar.
+  _origins_workflow_menu_creation_check($entity);
+}
+
+/**
+ * Check for menu creation.
+ */
+function _origins_workflow_menu_creation_check(EntityInterface $entity) {
+  // If a node is published using the moderation sidebar it is possible
+  // that menu creation might be overlooked. Code added here to rectify
+  // this bug.
+  $node_type = $entity->bundle();
+  $excluded_types = ['menu_link_content', 'content_moderation_state'];
```

Comment:
I'd pop this array directly into the in_array call instead of using up memory and put a comment above to explain.

Reply:
Fair enough, that's done now